packages/sync-actions/src/customer-actions.js

```diff
@@ -126,3 +141,61 @@ export function actionsMapShippingAddresses(diff, oldObj, newObj) {
 
   return handler(diff, oldObj, newObj)
 }
+
+export function actionsMapAuthenticationModes(diff, oldObj, newObj) {
+  // eslint-disable-next-line no-use-before-define
+  return buildAuthenticationModeActions({
+    actions: authenticationModeActionsList,
+    diff,
+    oldObj,
+    newObj,
+  })
+}
+
+function buildAuthenticationModeActions({ actions, diff, oldObj, newObj }) {
```

Comment:
had to build this modified version of `buildBaseAttributesActions` cause is a special case and this logic is only used for authentication modes.